publiq-ui/pub-label.stories.mdx

```diff
@@ -0,0 +1,34 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import PubLabel from './pub-label';
+
+<Meta title="Components/pub-label" component={PubLabel} />
+
+export const Template = (args, { argTypes }) => {
+  return {
+    props: Object.keys(argTypes),
+    components: { PubLabel },
+    template:
+      '<div><pub-label v-bind="$props">Label</pub-label><input type="text" id="custom-input"></div>',
+  };
+};
+
+## Default
+
+<Canvas>
+  <Story
+    name="Default"
+    args={{ id: 'custom-input' }}
+    argTypes={{ id: { control: { type: false } } }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+```javascript
+<div>
+  <pub-label id="custom-input">Label</pub-label>
```

Comment:
Nit: The label and input field stick together in the story. It's good that they don't have a pre-set margin, but maybe we can make this a bit prettier on the eyes by adding a `<br />` so the input is on a newline in this story specifically.